bridged-tokens-wrapper/src/lib.rs

```diff
@@ -73,6 +73,14 @@ pub trait BridgedTokensWrapper {
         chain_to_universal_mapper.clear();
     }
 
+    #[payable("*")]
+    #[endpoint(depositLiquidity)]
+    fn deposit_liquidity(&self) {
+        let (payment_amount, payment_token) = self.call_value().payment_token_pair();
+        self.token_liquidity(&payment_token)
```

Comment:
No particular checks?

Reply:
I don't think we have to do any checks, as there is an use-case to add liquidity for a token that is not whitelisted yet.